### ISSUE TITLE
Inner block doesn't work if i is remapped in visual mode

### ISSUE THREAD

**Issue by infokiller** [rhysd/vim-textobj-anyblock#13]
Hey,

I have weird vim mappings in which I remap the movement keys `hjkl` to `ijkl`, so that the geometry more closely matches that of the keyboard arrows. 
I noticed that with my mappings the inner block mappings don't work. I tried to debug a little with my very limited vimscript skills, and it seems that the function `get_region` doesn't work with my mappings. To reproduce, run the following vim command (before or after the plugin) and try the plugin inner block command: `xmap i k`

**Comment by rhysd** [comment]
Please use following configuration for your case.

```vim
let g:textobj#anyblock#inner_block_mapping = 'k'
```

**Comment by infokiller** [comment]
This doesn't seem to work for me. I'm testing it on the following file content:
```
a 'bcd'
efg
```
Then I'm invoking `dib` when I'm inside the single quotes, and it does something weird instead of deleting the content inside the quotes.

**Comment by infokiller** [comment]
So I realized your workaround assumes that I have something like `xmap k i` in addition to `xmap i k` (i.e. that I just switched `i` and `k`), but that's not the case. When I added `xmap k i` it does work.
The problem is that I don't have any single key mapped to `i` in visual mode- my replacement to `i` is `<space>i`, but the following doesn't work:
```
let g:textobj#anyblock#inner_block_mapping = '<space>i'
```
Does `g:textobj#anyblock#inner_block_mapping` only work for a single key?

My current workaround is to do:
```
xnoremap ♁ i
let g:textobj#anyblock#inner_block_mapping = '♁'
```
But it would actually be better if I could use a `<plug>` mapping here, i.e. if the following would work:
```
xnoremap <plug>(i)
let g:textobj#anyblock#inner_block_mapping = '<plug>(i)'
```
However, this doesn't work for me, only the workaround above with `♁` (or any other single unicode char).
